_redirects

```diff
@@ -131,7 +132,7 @@
 
 /docs/ftw-testing-error-handling/how-to-use-ci-with-ftw/ /docs/ftw/how-to-use-ci-with-ftw/
 /docs/ftw-styling/how-to-change-ftw-icons/ /docs/ftw/how-to-change-ftw-icons/
-/docs/ftw-styling/how-to-change-ftw-ui-texts-and-translations/ /docs/ftw/how-to-change-ftw-ui-texts-and-translations/
+/docs/ftw-styling/how-to-change-ftw-ui-texts-and-translations/ /docs/ftw/how-to-change-ftw-ui-texts/
```

Comment:
In addition to this: "how to add static pages in FTW" needs redirection

Reply:
In the reorganisation project we updated the URLs so that subcategories are no longer a part of the URL, i.e. changing an article's category doesn't change its url.